DataFormats/GeometryVector/interface/Phi.h

```diff
@@ -21,13 +21,14 @@ namespace Geom {
   using angle_units::operators::operator""_deg;
   using angle_units::operators::convertRadToDeg;
 
-  enum class PhiRange {MinusPiToPi, ZeroTo2pi};
+  struct MinusPiToPi {};  // Dummy struct to indicate -pi to pi range
+  struct ZeroTo2pi{};  // Dummy struct to indicate 0 to 2pi range
```

Comment:
You could enclose these into another struct if you'd like to keep `PhiRange::MinusPiToPi` etc
```cpp
struct PhiRange {
  struct MinusPiToPi {};  // Dummy struct to indicate -pi to pi range
  struct ZeroTo2pi{};  // Dummy struct to indicate 0 to 2pi range
};
```

Reply:
Thanks for the suggestion, but I don't really want to create another dummy `struct`. I'll just stick with these two.
I just learned from Philippe Canal that genreflex cannot correctly handle an `enum class` as a non-type template parameter, so my previous approach could not work.